Anguandia/quickcredit
# Ft db view current loans 166115442

#### Purpose of this pull request

Add admin view selection of loans from database

#### Tasks to be accomplished

- Add URL query string pursing for collection selection key and target values
- Maintain selection of all if no query

#### How to test these features manually

- With a local copy of this application populate the database with a variety of loans, creating some with repaid=true; others status approved or rejected or the default pending or repaid, etc
- With different combinations of status/status-repaid query parameter values within the permitted, make GET/loans?query requests and notice the filtering of the response items

Pivotal tracker stories affected

[finishes #166115442 #166115468 #166229490]
